rest/data/repotracker.go

```diff
@@ -52,7 +53,17 @@ func (c *RepoTrackerConnector) TriggerRepotracker(q amboy.Queue, msgID string, e
 		})
 		return errors.New("repotracker is disabled")
 	}
-
+	if len(settings.GithubOrgs) > 0 && !util.StringSliceContains(settings.GithubOrgs, *event.Repo.Owner.Name) {
+		grip.Error(message.WrapError(err, message.Fields{
```

Comment:
this will never log, because it's the err (which is nil from line 40), and WrapError only logs if the err is non-nil

You should probably also return early? 

Also grip.ErrorWhen(<cond>, message.Fields{}) is a good option for this if <log>.

Reply:
I'd prefer not to grip.ErrorWhen, I think it's not as clear to read.